data/data/bootstrap/files/usr/local/bin/bootkube.sh.template

```diff
@@ -112,8 +112,8 @@ then
 		/usr/bin/cluster-kube-apiserver-operator render \
 		--manifest-etcd-serving-ca=etcd-ca-bundle.crt \
 		--manifest-etcd-server-urls={{.EtcdCluster}} \
-		--manifest-image=${OPENSHIFT_HYPERKUBE_IMAGE} \
-		--manifest-operator-image=${KUBE_APISERVER_OPERATOR_IMAGE} \
+		--manifest-image="${OPENSHIFT_HYPERKUBE_IMAGE}" \
```

Comment:
does it matter? the image name (pull spec) is already restrictive such that we shouldn't care right?

Reply:
> the image name (pull spec) is already restrictive such that we shouldn't care right?

Right, there should be no shell-sensitive characters in the pullspec.  But quoting means we don't need to rely on that or worry about what happens if our pullspec-extraction commands earlier in this script happen to spit out something that does contain whitespace.  And it hushes up ShellCheck so we can see if there are any other issues with the template that _are_ actually problems.